apps/spruce/src/pages/waterfall/VersionLabel/index.tsx

```diff
@@ -31,15 +42,17 @@ export const VersionLabel: React.FC<VersionFields> = ({
 
   const { sendEvent } = useWaterfallAnalytics();
 
+  const commitType = activated ? "active" : "inactive";
```

Comment:
It would probably be cleaner to case directly on `activated` rather than introduce a magic string

Reply:
Hmm good point, I'm okay with either approach. Your call!